crypto/nrf_cc310_platform/CHANGELOG.rst

```diff
@@ -447,24 +449,23 @@ Added a new build of nrf_cc3xx_mbedcrypto libraries for nRF9160, nRF52840, and n
     * ``cortex-m4/soft-float/short-wchar/no-interrupts/libnrf_cc310_platform_0.9.11.a``
     * ``cortex-m4/hard-float/short-wchar/no-interrupts/libnrf_cc310_platform_0.9.11.a``
 
-
 nrf_cc3xx_platform - 0.9.10
 ***************************
 
-New version of the library with an added feature:
+New version of the library with a new feature:
 
-* Added HMAC_DRBG APIs in :file:`nrf_cc3xx_platform_hmac_drbg.h`
+* Added HMAC_DRBG APIs in :file:`nrf_cc3xx_platform_hmac_drbg.h`.
```

Comment:
Why period?

Reply:
Why not?

Reply:
Just having the RMF page 66 in mind, and thinking this really is not a complete sentence but a fragment instead.
We might need aligning with these rules, but I leave it up to you to decidw if this is worth spending your resources on for the moment.

Reply:
Good point, indeed it is a fragment.